samples/sensor/bmi160/sample.yaml

```diff
@@ -1,8 +1,7 @@
 sample:
-  description: TBD
-  name: TBD
+  name: BMI160 Sensor Sample
 tests:
   test:
-    build_only: true
-    platform_whitelist: arduino_101_sss
+    harness: sensor
     tags: sensors
+    depends_on: spi gpio
```

Comment:
curious about the filtering.
we are saying that this depends on SPI and GPIO, but how do we know a bmi160 sensor is present on the board to run the test?

Reply:
based on recent developments related to DTS that are in the tree now, the sensors on a board will be defined in the board DTS, otherwise, if the sensor is off board, it will be matches based on a harness with that sensor, right now the harness is generic and called sensor.